src/main/java/webserver/controller/GetUserListController.java

```diff
@@ -0,0 +1,83 @@
+package webserver.controller;
+
+import model.Users;
+import utils.TemplateCreator;
+import webserver.http.header.HttpCharacterEncoding;
+import webserver.http.header.HttpHeader;
+import webserver.http.header.HttpHeaderName;
+import webserver.http.header.HttpHeaders;
+import webserver.http.message.HttpRequestMessage;
+import webserver.http.message.HttpResponseMessage;
+import webserver.http.request.HttpResourceType;
+import webserver.http.response.HttpStatus;
+import webserver.http.session.HttpSessionFinder;
+import webserver.service.UserService;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static webserver.http.header.HttpHeaders.HEADER_VALUE_DELIMITER;
+
+public class GetUserListController implements Controller {
+
+    @Override
+    public HttpResponseMessage createHttpResponseMessage(HttpRequestMessage httpRequestMessage) {
+        Optional<String> cookieValue = httpRequestMessage.getHeaderValue(HttpHeaderName.COOKIE.getName());
```

Comment:
```suggestion
        return httpRequestMessage.getHeaderValue(HttpHeaderName.COOKIE.getName())
                .filter(this::isLogined)
                .map(cookie -> createUserListHttpResponseMessage())
                .orElse(createNonUserHttpResponseMessage());
```
optional을 이런식으로 활용 할수도 있으니 참고하세요